kernel/classes/ezcontentobjectattribute.php

```diff
@@ -1182,7 +1182,7 @@ static function metaDataArray( &$attributes, $skipRelationAttributes = false )
             }
 
             $datatype = $attribute->dataType();
-            if($skipRelationAttributes && $datatype->isRelationType())
+            if( $skipRelationAttributes && $datatype->isRelationType() )
```

Comment:
my prev comment forgot to also specify whitespace after `if`

Reply:
Fixed.